src/test/file-utils.test.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const fileUtils = require('../main/lib/file-utils');
+
+describe('file-utils', () => {
+
+  describe('listFilesInDirectory', () => {
+    it('lists all files in the directory if no filters are provided', () => {
+      let filesInDirectory = fileUtils.listFilesInDirectory(
```

Comment:
Replace `let` with `const` please. Variable is never reassigned not need to use `let`.

Reply:
Done